examples/graph/wgan-graphs.py

```diff
@@ -0,0 +1,601 @@
+"""
+Title: WGAN-GP with R-GCN for the generation of small molecular graphs
+Author: [akensert](https://github.com/akensert)
+Date created: 2021/06/28
+Last modified: 2021/06/28
+Description: Complete implementation of WGAN-GP with R-GCN to generate novel molecules.
+"""
+
+"""
+## Introduction
+
+In this tutorial, a generative model for graphs will be implemented to generate
```

Comment:
Prefer using the active voice: "we implement a generative model for graphs and use it to generate new molecule."